src/app/zap-templates/zcl/data-model/chip/chip-types.xml

```diff
@@ -0,0 +1,97 @@
+    <type id="0x19" description="16-bit bitmap"            name="bitmap16"          size="2"  discrete="true" />
+    <type id="0x1A" description="24-bit bitmap"            name="bitmap24"          size="3"  discrete="true" />
+    <type id="0x1B" description="32-bit bitmap"            name="bitmap32"          size="4"  discrete="true" />
+    <type id="0x1C" description="40-bit bitmap"            name="bitmap40"          size="5"  discrete="true" />
+    <type id="0x1D" description="48-bit bitmap"            name="bitmap48"          size="6"  discrete="true" />
+    <type id="0x1E" description="56-bit bitmap"            name="bitmap56"          size="7"  discrete="true" />
+    <type id="0x1F" description="64-bit bitmap"            name="bitmap64"          size="8"  discrete="true" />
+    <type id="0x20" description="Unsigned 8-bit integer"   name="int8u"             size="1"  analog="true"   />
+    <type id="0x21" description="Unsigned 16-bit integer"  name="int16u"            size="2"  analog="true"   />
+    <type id="0x22" description="Unsigned 24-bit integer"  name="int24u"            size="3"  analog="true"   />
+    <type id="0x23" description="Unsigned 32-bit integer"  name="int32u"            size="4"  analog="true"   />
+    <type id="0x24" description="Unsigned 40-bit integer"  name="int40u"            size="5"  analog="true"   />
+    <type id="0x25" description="Unsigned 48-bit integer"  name="int48u"            size="6"  analog="true"   />
+    <type id="0x26" description="Unsigned 56-bit integer"  name="int56u"            size="7"  analog="true"   />
+    <type id="0x27" description="Unsigned 64-bit integer"  name="int64u"            size="8"  analog="true"   />
+    <type id="0x28" description="Signed 8-bit integer"     name="int8s"             size="1"  analog="true"   />
+    <type id="0x29" description="Signed 16-bit integer"    name="int16s"            size="2"  analog="true"   />
+    <type id="0x2A" description="Signed 24-bit integer"    name="int24s"            size="3"  analog="true"   />
+    <type id="0x2B" description="Signed 32-bit integer"    name="int32s"            size="4"  analog="true"   />
+    <type id="0x2C" description="Signed 40-bit integer"    name="int40s"            size="5"  analog="true"   />
+    <type id="0x2D" description="Signed 48-bit integer"    name="int48s"            size="6"  analog="true"   />
+    <type id="0x2E" description="Signed 56-bit integer"    name="int56s"            size="7"  analog="true"   />
+    <type id="0x2F" description="Signed 64-bit integer"    name="int64s"            size="8"  analog="true"   />
+    <type id="0x30" description="8-bit enumeration"        name="enum8"             size="1"  discrete="true" />
+    <type id="0x31" description="16-bit enumeration"       name="enum16"            size="2"  discrete="true" />
+    <type id="0x38" description="Semi-precision"           name="float_semi"        size="2"  analog="true"   />
```

Comment:
This semi-precision type doesn't exist in Matter, fwiw.

Reply:
> This semi-precision type doesn't exist in Matter, fwiw.

I have tried to keep all the types from ZCL at the moment. There are many types that does not exists in Matter and will be removed in #7873